### DISCUSSION TITLE
cnc 3018 30w lazer çalışmıyor

### DISCUSSION THREAD

**Discussion by Abdulbasit-Tutus** [arkypita/LaserGRBL#1279] [Q&A]
Merhabalar... Programı 0.9 dan 1.1f sürümüne yükseltemedim öncelikle bununla ilgili yardım istiyorum. ayrıca $32=0 ve $32=1 komutları lazeri açmıyor.. Lazer test tuşuna bastığımda çalışıyor fakat kesim yapacağım zaman lazer açılmıyor. sanki lazere güç gelmiyor bunun yerine iş miline güç gidiyor. sonuç olarak lazerim açılmıyor kesim ya da çizim yapamıyorum bununla ilgili yardımlarınızı bekliyorum... şimdiden teşekkür ederim..

![image](https://user-images.githubusercontent.com/77171881/104289673-7b147f80-54ca-11eb-997c-036972409bb5.png)

![image](https://user-images.githubusercontent.com/77171881/104290140-0a219780-54cb-11eb-8f07-3ff41ec880d6.png)

**Comment by Mrjavaci** [comment]
0.9 'dan 1.1F'e güncellemen gereken program laserGRBL değil. Güncellemen gereken program GRBL.
Arduino'nu driverları yüklü bir bilgisayara taktıktan sonra ilgili hex dosyasını indirip ( https://github.com/gnea/grbl/releases/download/v1.1h.20190825/grbl_v1.1h.20190825.hex ) ardından https://github.com/xinabox/xLoader ile flash'laman lazım. 

Sonrasında LaserGRBL 'de M komutlarını ayarlaman gerekebiliyor. Arduino üzerindeki GRBL'ı güncelledikten sonra problem devam ediyorsa tekrar yazarsın.

Ayrıca Github'da ingilizce destek almaya çalışırsan daha hızlı alabilirsin. 

İyi çalışmalar.
